app/scripts/proactive/templates/catalog-publish.html

```diff
@@ -28,6 +28,12 @@
 						<div class="pull-left" id="catalog-publish-objects-title" style="font-weight: bold;padding-left: 15px;">Objects</div>
 					</legend>
 					<div style="height: 270px; overflow: auto;">
+						<form class="container-flex w-100" id="new-script-form">
+							<div class="input-group w-100">
+								<input id="new-script-name" type="text" placeholder="new script name" class="form-control" style="display: table-cell">
+								<button id="add-new-script" class="btn btn-default"><i class="fa fa fa-plus"></i></button>
```

Comment:
do you still need `id="add-new-script"`  ?

you should add `type="submit"` to the button